core/tests/webdriverio_utils/ExplorationEditorSettingsTab.js

```diff
@@ -102,31 +115,40 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.setCategory = async function(category) {
+    await action.click('Neutral element', neutralElement);
```

Comment:
Why is this added? Also, it's not being added to the protractor code.


More broadly, I'm seeing a lot of added neutral element clicks, but I don't understand why they're needed

Reply:
removed